### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by panianchuk**
https://panianchuk.github.io/react_todo-app-with-api/

### CODE REVIEW

File: src/api/todos.ts [hunk 1]
```diff
@@ -5,4 +5,18 @@ export const getTodos = (userId: number) => {
...
+export const deleteTodo = (id: number) => {
+  return client.delete(`/todos/${id}`);
+};
+
+export const updateTodo = (id: number, data: boolean | string) => {
+  if (typeof data === 'boolean') {
+    return client.patch<Todo>(`/todos/${id}`, { completed: data });
+  }
+
+  return client.patch<Todo>(`/todos/${id}`, { title: data });
```
Comment (loralevitska) [inline src/api/todos.ts:21]: It is better to separate the logic for such things, or use objects where you overwrite the necessary data

File: src/components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  setTitle: (title: string) => void;
+  onAddToto: (title: string) => void;
+  isLoading: boolean;
+  completedAllTodo: () => void;
+  allCompleted: boolean;
+};
+
+type Ref = HTMLInputElement;
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
```
Comment (loralevitska) [inline src/components/Header.tsx:15]: Try to fix this rule and remove comments. Fix all cases if present

File: src/components/TodoList.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          toggleTodo={toggleTodo}
+          // ============================
+          isRenamingTodo={isRenamingTodo}
+          setIsRenamingTodo={setIsRenamingTodo}
+          selectedTodoById={selectedTodoById}
+          dblClickHandler={dblClickHandler}
+          updateTitleTodo={updateTitleTodo}
+        />
+      ))}
+    </section>
```
Comment (loralevitska) [inline src/components/TodoList.tsx:45]: Use semantic tags for list `ul` `li`
And remove comment

**Review by loralevitska — CHANGES_REQUESTED** [review]
Keep up the good work! Some comments to fix:
Remove default outline 
image
image

File: src/components/Header.tsx (hunk 2 above)
Comment (loralevitska) [reply src/components/Header.tsx:15]: still not fixed

File: src/api/todos.ts (hunk 1 above)
Comment (loralevitska) [reply src/api/todos.ts:21]: not fixed

**Review by loralevitska — CHANGES_REQUESTED** [review]
Hi, I see you haven't fixed previous comments, maybe you did not push your changes? If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**